### DISCUSSION TITLE
Ran out of space, lost the Spacemesh program directory, now unable to use my PoST data

### DISCUSSION THREAD

**Discussion by 1472583610** [spacemeshos/go-spacemesh#6373] [General]
I ran out of space on the drive where my Spacemesh is installed and where the node data is. Spacemesh crashed and the node configuration file ended up completely empty. Spacemesh couldn't be brough back to life, there were unhandled exceptions upon launch.

Reinstallation of Spacemesh resulted in loss of all node data. Now, when I try to reconnect my PoST data, I'm unable to do so as Spacemesh reports a mismatch between configurations.

Is there any way out of this or are my 6 TBs of PoST data useless now?

**Comment by poszu** [comment]
Hi @1472583610, I'm sorry to hear that. There might be ways to recover, but I need more information to help you. First, please clarify "node configuration file" ended up empty. Do you still have your key file? It's typically a file located in spacemesh/identities/local.key. If this file was overwritten during the reinstallation of Smapp and you don't have a backup, then I'm afraid nothing can be done.

**Reply by 1472583610** [reply]
> It's typically a file located in spacemesh/identities/local.key.

Hi poszu, thank you very much, that was the information I was missing. It seems to be working now. Thankfully I did have a backup of the entire node-data directory.

To provide the clarification, I meant the node-config.json file located in AppData/Roaming/Spacemesh folder. After the space ran out, this config file was empty (contained 0 bytes), so I guess the space ran out when the file was being overwritten.

Again, thank you very much for your help.
